src/surface.lisp

```diff
@@ -11,8 +11,12 @@
   (c-ref surface sdl2-ffi:sdl-surface :pixels))
 
 (defun surface-format (surface)
+  (c-ref surface sdl2-ffi:sdl-surface :format))
+
+(defun surface-format-format (surface)
   (enum-key '(:enum (sdl-pixel-format))
-            (c-ref surface sdl2-ffi:sdl-surface :format :format)))
+            (c-ref (c-ref surface sdl2-ffi::sdl-surface :format)
+                   sdl2-ffi::sdl-pixel-format :format)))
 
```

Comment:
Multiple things took the pointer in the tutorial I was following, which is why I added both.

I take it the latest version of cl-autowrap isn't in quicklisp?